ivy/functional/frontends/torch/tensor.py

```diff
@@ -1088,6 +1088,9 @@ def atan2_(self, other):
     def fmin(self, other):
         return torch_frontend.fmin(self, other)
 
+    def msort(self, out=None):
```

Comment:
I think the out is not needed here

Reply:
Removed in the latest commit, thanks for spotting this out (pun)!